app/celery/letters_pdf_tasks.py

```diff
@@ -228,16 +229,27 @@ def send_letters_volume_email_to_dvla(letters_volumes, date):
         send_notification_to_queue(saved_notification, queue=QueueNames.NOTIFY)
 
 
-def send_dvla_letters_via_api(print_run_deadline_local):
+def send_dvla_letters_via_api(print_run_deadline_local, batch_size=100):
     current_app.logger.info("send-dvla-letters-for-day-via-api - starting queuing")
-    for i, row in enumerate(dao_get_letters_to_be_printed(print_run_deadline_local)):
-        if i % 10000 == 0:
-            current_app.logger.info("triggered tasks for %i letters", i)
-        deliver_letter.apply_async(kwargs={"notification_id": row.id}, queue=QueueNames.SEND_LETTER)
+    for batch in batched(
+        (row.id for row in dao_get_letters_to_be_printed(print_run_deadline_local)),
+        batch_size,
+    ):
+        current_app.logger.info("triggered tasks for %i letters", batch_size)
```

Comment:
this'll log a lot (100x more than it logs atm!). i think we don't really need this as we'll still have the "send-dvla-letters-for-day-via-api - finished queuing" log below (tbh we can remove that log line too and just rely on the normal "Celery task x took y seconds" log line) as the collate task doesn't do anything after triggering all these, and just finishes
```suggestion
```

Reply:
f782fe2